browser/ui/BUILD.gn

```diff
@@ -622,6 +626,13 @@ source_set("ui") {
     ]
   }
 
+  if (enable_request_otr) {
+    deps += [
+      "//brave/components/request_otr/browser",
+      "//brave/components/request_otr/common",
```

Comment:
common is inside `enable_request_otr` guard here, but not  https://github.com/brave/brave-core/pull/17365/files#diff-7ce4cface9f839816f0fc02bad0e04aa80284b048ef0d8f0876547e6e37e38b6R199
I think you probably want `assert(enable_request_otr)` in common?

Reply:
I'm not opposed to leaving common outside the buildflag, but then it should be outside the buildflag everywhere in gn and cc

Reply:
added assert and guard